www/src/layouts/shared/Header/Header.tsx

```diff
@@ -15,31 +14,13 @@ interface INavigationProps {
 
 const Navigation = ({ mobile }: INavigationProps): JSX.Element => (
   <Stack as="nav" isInline={!mobile} spacing={mobile ? `20` : `3`}>
-    <Button
-      variant="ghost"
-      fontSize={mobile ? `2xl` : `1.125rem`}
-      as={Link}
-      to={`/about`}
-      fontFamily={`'Inter', ui-serif, ${defaultFontBody}`}
-    >
+    <Button variant="ghost" fontSize={mobile ? `2xl` : `md`} as={Link} to={`/about`}>
       ABOUT
     </Button>
-    <Button
-      variant="ghost"
-      fontSize={mobile ? `2xl` : `1.125rem`}
-      as={Link}
-      to={`/portfolio`}
-      fontFamily={`'Inter', ui-serif, ${defaultFontBody}`}
-    >
+    <Button variant="ghost" fontSize={mobile ? `2xl` : `md`} as={Link} to={`/portfolio`}>
       PORTFOLIO
     </Button>
-    <Button
-      variant="ghost"
-      fontSize={mobile ? `2xl` : `1.125rem`}
-      as={Link}
-      to={`/writing`}
-      fontFamily={`'Inter', ui-serif, ${defaultFontBody}`}
-    >
+    <Button variant="ghost" fontSize={mobile ? `2xl` : `md`} as={Link} to={`/writing`}>
```

Comment:
Buttons may not be the appropriate tag to use for header links... at least they're not often used in other sites.

Reply:
Possibly redo the whole header in a restructure.